help/legacy/_posts/2019-10-25-icrcstatus.md

```diff
@@ -0,0 +1,337 @@
+---
+layout: post
+title: "Status of the ICRC series in the ADS"
+order: 3
+---
+Status of the ICRC series in the ADS
```

Comment:
Different styling than the other posts, so the space isn't an issue, but the text duplication may be